lit/index.lit

```diff
@@ -13,25 +13,27 @@
   \link{CI}{https://en.wikipedia.org/wiki/Continuous_integration}/\link{CD}{https://en.wikipedia.org/wiki/Continuous_delivery}.
 }
 
-\quick-start{
+\right-side{Quick Start}{
```

Comment:
Wondering if we could perhaps make all sidebar code blocks use the black? the grey on grey might be hard to distinguish on lower color-quality monitors.

Reply:
done